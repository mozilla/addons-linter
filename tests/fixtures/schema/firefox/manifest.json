[
  {
    "namespace": "manifest",
    "permissions": [],
    "types": [
      {
        "id": "Permission",
        "choices": [
          {
            "type": "string",
            "enum": [
              "alarms",
              "clipboardRead",
              "clipboardWrite",
              "geolocation",
              "idle",
              "notifications",
              "storage"
            ]
          },
          { "$ref": "MatchPattern" }
        ]
      }
    ]
  }
]
