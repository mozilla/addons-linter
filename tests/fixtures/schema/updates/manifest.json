{
  "cookies": {
    "types": {
      "Cookie": {
        "properties": {
          "name": {"pattern": "\\w+"}
        }
      }
    }
  }
}
