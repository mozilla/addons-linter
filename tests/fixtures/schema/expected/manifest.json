{
  "$id": "manifest",
  "permissions": [],
  "definitions": {},
  "refs": {},
  "types": {
    "Permission": {
      "anyOf": [
        {
          "type": "string",
          "enum": [
            "alarms",
            "clipboardRead",
            "clipboardWrite",
            "geolocation",
            "idle",
            "notifications",
            "storage"
          ]
        },
        { "$ref": "#/types/MatchPattern" },
        { "$ref": "cookies#/definitions/Permission" }
      ]
    }
  }
}
